templates/sso-cd-x509-https.json

```diff
@@ -369,6 +388,12 @@
                                 "secret": {
                                     "secretName": "sso-x509-jgroups-secret"
                                 }
+                            },
+                            {
+                                "name": "service-ca",
+                                "configMap": {
+                                    "configMapName": "${APPLICATION_NAME}-service-ca"
```

Comment:
Replace `"configMapName"` with `"name"` like in RH-SSO 7.3 & 7.4 case

Reply:
Done